src/auth/index.js

```diff
@@ -314,20 +444,20 @@ export default class Auth {
     const payload = apply(
       {
         parameters: {
-          email: { required: true },
-          password: { required: true },
-          connection: { required: true },
-          username: { required: false },
-          metadata: { required: false, toName: 'user_metadata' }
-        }
+          email: {required: true},
+          password: {required: true},
+          connection: {required: true},
+          username: {required: false},
+          metadata: {required: false, toName: 'user_metadata'},
+        },
       },
-      parameters
+      parameters,
```

Comment:
Do the style conventions for this repo allow trailing commas?

Reply:
Yes, enforced via pre-commit hook https://github.com/auth0/react-native-auth0/blob/master/.prettierrc.js#L5